ops/pebble.py

```diff
@@ -2153,11 +2153,15 @@ def remove_path(self, path: str, *, recursive: bool = False):
 
         Args:
             path: Path of the file or directory to delete from the remote system.
-            recursive: If True, and path is a directory recursively deletes it and
+            recursive: If True, and path is a directory, recursively deletes it and
```

Comment:
Can we also please copy the new version above to here? "recursive" and "Raises".